prometheus/generic/setup-netapp-exporter/README.md

```diff
@@ -0,0 +1,67 @@
+# setup-netapp-exporter
+
+This role will instantiate a NetApp health exporter container on targeted hosts.
+
+## Requirements
+
+Docker must be available and running on the targeted hosts.
+
+## Role Variables
+
+Default values of variables:
+
+```
+netapp_exporter_image: 'quay.io/redhat-cop/monitoring-netapp-exporter'
+netapp_exporter_image_version: 'latest'
+netapp_exporter_port: '9418'
+
+provision_state: "started"
+```
+
+`netapp_exporter_image` - The NetApp exporter image to deploy.
+
+`netapp_exporter_image_version` - The image tag to deploy.
+
+`netapp_exporter_port` - The port to expose on the target hosts.
+
+`provision_state` - Options: [absent, killed, present, reloaded, restarted, **started** (default), stopped]
+
+## Example Inventory
+
+Snippet below configures targets for the NetApp exporter.
+
+```
+monitoring_username: "Administrator"
+monitoring_password: "password123"
```

Comment:
same comment as the Flex PR:
https://github.com/redhat-cop/monitoring/pull/45#pullrequestreview-654768379